sqldb/postgres_test.go

```diff
@@ -7,6 +7,11 @@ import (
 	"testing"
 )
 
+// isSQLite is false if the build tag is set to test_db_postgres. It is used in
+// tests that compile for both SQLite and Postgres databases to determine
+// which database implementation is being used.
```

Comment:
Can we add a TODO here that with `sqldbv2` there will be a `DatabaseType` on the `BaseDB` struct that will allow us to query this at runtime.